resources/styles/components/task/index.less

```diff
@@ -14,12 +14,19 @@
 .concept-coach-title {
   margin-bottom: 1.5rem;
 
-  .chapter-section-prefix,
-  .back-to-book {
+  .chapter-section-prefix {
     display: inline-block;
     margin-right: 1rem;
   }
 
+  &.back-to-book {
+    font-size: 2.4rem;
+
+    span {
```

Comment:
hrm, I'm guessing this is a `react-bootstrap` element (so no class?) but could/should this be prefixed with a `>` to reduce bleed?